lib/processor.js

```diff
@@ -16,7 +16,7 @@ const SUPPORTS_AUTOFIX = true;
 
 const markdown = unified().use(remarkParse);
 
-let blocks = [];
```

Comment:
The `blocks` will not work when writing markdown codes in code blocks.

This is not a problem because there is no `markdown -> md` mapper before, so `.markdown` files are not linted actually.